auditorium/src/views/components/auditorium/format.js

```diff
@@ -19,18 +20,21 @@ const Format = (props) => {
       break
     case 'count':
       value = formatCount(source)
+      tooltip = value !== Math.floor(source)
       break
     case 'percentage':
       value = formatNumber(source, 100, 1) + ' %'
       break
     default:
       value = formatNumber(source, factor, digits)
   }
-  return (
-    <Fragment>
-      {value}
-    </Fragment>
-  )
+  return tooltip
+    ? (
+      <span title={source}>
```

Comment:
Do you think it would make sense to format the value passed to the `title` attribute as well? E.g. when working with `234543`, the component would render `234.5k` and a title of `234543` when we _could_ also display `234.5k` and a title of `234.543` for easier understanding of large numbers like this. A `formatNumber` helper that does this for us is available in the same file already.

----

Important caveat: there seems to be a pre-existing bug (of course...) in the `formatNumber` helper where it right now wouldn't allow passing `0` as a value for the `digits` parameter as it's falsy and `digits || 1` would evaluate to `1` in this case. So this would need to be fixed before the above suggestion could be added.

Reply:
Good suggestion. It makes sense to format the value. 

I fixed the bug in `formatNumber` and added formatting to the value passed to the `title` attribute. Let me know if the solution is ok.

Reply:
Looks brilliant, the drive by fix is much appreciated.